tests/microos/image_checks.pm

```diff
@@ -18,6 +18,9 @@ sub run {
 
     # Disk which /var resides on
     my $disk = script_output 'lsblk -rnoPKNAME $(findmnt -nrvoSOURCE /var)';
+    if (!$disk && check_var('FLAVOR', 'Default-encrypted')) {
+        $disk = (split('/', script_output 'blkid -l -t TYPE="crypto_LUKS" -o device'))[-1];
```

Comment:
This will return the partition but it needs to return the disk.

Reply:
oh, true, fixing!